soaks/bin/soak_one.sh

```diff
@@ -18,14 +19,13 @@ display_usage() {
     echo "  --variant: the variation of test in play, either 'baseline' or 'comparison'"
     echo "  --tag: the tag this test covers"
     echo "  --capture-dir: the directory in which to write captures"
-    echo "  --cpus: the total number of CPUs to dedicate to the soak minikube, default 7"
-    echo "  --memory: the total amount of memory dedicate to the soak minikube, default 8g"
     echo "  --vector-cpus: the total number of CPUs to give to soaked vector"
-    echo "  --warmup-seconds: the total number seconds to pause waiting for vector to warm up"
+    echo "  --replicas: the total number of replica experiments to run, default 3"
     echo ""
 }
 
 BUILD_IMAGE="true"
+REPLICAS=2
```

Comment:
This doesn't seem to match the documented default of `3`. I might also keep the responsibility of running multiple replicas up in `soak.sh` and let this script just be for running one like the name implies.

Reply:
Ah, the "one" is with regard to the variant. But, yes, this is a bug.

Reply:
Ah, yes, maybe the script should be called `soak_variant.sh` then?

Reply:
Yeah, I buy that.

Reply:
I think, considering how large this PR is, I'd prefer to make this adjustment in a follow-up. Or possibly merge the two scripts together with regard to https://github.com/vectordotdev/vector/pull/11891#discussion_r831536322.